src/components/bottom_section/Buttons.js

```diff
@@ -135,18 +131,12 @@ const Buttons = ({
         operation={operation}
         waitingValue={waitingValue}
         displayResult={displayResult}
-        setOperation = {setOperation}
+        setOperation={setOperation}
       />
 
-      <button className="button" onClick={() => getButtonValue(4)}>
-        4
-      </button>
-      <button className="button" onClick={() => getButtonValue(5)}>
-        5
-      </button>
-      <button className="button" onClick={() => getButtonValue(6)}>
-        6
-      </button>
+      <input type="button" value={4} onClick={handleButtonClick} />
+      <input type="button" value={5} onClick={handleButtonClick} />
+      <input type="button" value={6} onClick={handleButtonClick} />
```

Comment:
Thats a perfect example how to ruin an app's accessibility, using correct semantic elements helps to make sure that page provides best user experience, you should never use div tags for all elements, inputs where not needed, selects for phone numbers, etc